UnityProject/Assets/Scripts/Electricity/PowerSupplies/PowerGenerator.cs

```diff
@@ -154,8 +154,7 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 	{
 		if (!DefaultWillInteract.Default(interaction, side)) return false;
 		if (interaction.TargetObject != gameObject) return false;
-		if (interaction.HandObject != null &&
-				!Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.SolidPlasma)) return false;
+		if (!Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.SolidPlasma)) return false;
```

Comment:
Won't this mean you will not be able to toggle the power state of the generator?